_data/nav.yml

```diff
@@ -234,6 +234,9 @@
       - url: "#API"
     - url: "advcode"
     - url: "advtable"
+      pages:
+      - url: "#Events"
```

Comment:
My bad, should have spotted this earlier, but can you move the events section to after the commands section?